relbench/datasets/trial.py

```diff
@@ -0,0 +1,271 @@
+import os
+
+import pandas as pd
+import numpy as np
+import pooch
+
+from relbench.data import Database, RelBenchDataset, Table
+from relbench.tasks.trial import OutcomeTask, AdverseEventTask, WithdrawalTask, SiteSuccessTask, SponsorConditionTask
+from relbench.utils import unzip_processor
+
+
+class TrialDataset(RelBenchDataset):
+    name = "rel-trial"
+    # 2 years gap
+    val_timestamp = pd.Timestamp("2019-01-01")
+    test_timestamp = pd.Timestamp("2021-01-01")
```

Comment:
If we were to use 2 years gap here, our timedelta should not be larger than 2 years.